nixos/modules/rename.nix

```diff
@@ -169,6 +169,14 @@ with lib;
     # KDE Plasma 5
     (mkRenamedOptionModule [ "services" "xserver" "desktopManager" "kde5" ] [ "services" "xserver" "desktopManager" "plasma5" ])
 
+    # Qt
+    (mkRemovedOptionModule [ "qt5" "style" ] "Please use <option>qt5.theme</option> instead.")
```

Comment:
but it seems qt ubiquitously refers to them as styles?

Reply:
@worldofpeace Fixed, thanks!